src/cdk-experimental/drag-drop/drop.ts

```diff
@@ -0,0 +1,193 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  Component,
+  ContentChildren,
+  forwardRef,
+  Input,
+  ViewEncapsulation,
+  ChangeDetectionStrategy,
+  Output,
+  EventEmitter,
+  ElementRef,
+  QueryList,
+} from '@angular/core';
+import {CdkDrag} from './drag';
+import {CdkDragExit, CdkDragEnter, CdkDragDrop} from './drag-events';
+import {CDK_DROP_CONTAINER} from './drop-container';
+
+
+/** Container that wraps a set of draggable items. */
+@Component({
+  moduleId: module.id,
+  selector: 'cdk-drop',
+  exportAs: 'cdkDrop',
+  template: '<ng-content></ng-content>',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  styleUrls: ['drop.css'],
+  providers: [
+    {provide: CDK_DROP_CONTAINER, useExisting: CdkDrop},
+  ],
+  host: {
+    'class': 'cdk-drop',
+    '[class.cdk-drop-dragging]': '_dragging'
+  }
+})
+export class CdkDrop<T = any> {
+  /** Draggable items in the container. */
+  @ContentChildren(forwardRef(() => CdkDrag)) _draggables: QueryList<CdkDrag>;
+
+  /**
+   * Other draggable containers that this container is connected
+   * to and into which the container's items can be transferred.
+   */
+  @Input('connectedTo') siblings: CdkDrop[] = [];
```

Comment:
I think it's a little confusing for the input and the property to be so different